### PR TITLE
Feature html css

### PR DESCRIPTION
**Pull request by sammysosa23**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,29 @@
-# Project1
```
Comment (sammysosa23) [inline README.md]: ignore

File: css/styleProject.css [hunk 2]
```diff
@@ -1,8 +1,8 @@
 html, body {
-    height: 95%;
```
Comment (sammysosa23) [inline css/styleProject.css]: well keep the 100%

File: css/styleProject.css [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
+    height: 100%;
 }
 #cover {
-    background: #222 url("../images/jupiter.jpg") center center no-repeat;
```
Comment (sammysosa23) [inline css/styleProject.css]: ignore

File: css/styleProject.css [hunk 4]
```diff
@@ -17,22 +17,137 @@ html, body {
     align-items: center;
 
 }
-#trendingButtons.absolute {
```
Comment (sammysosa23) [inline css/styleProject.css]: ignore

File: css/styleProject.css [hunk 5]
```diff
@@ -17,22 +17,137 @@ html, body {
...
-    position: absolute;
-    top: 50px;
-    right: 50px;
+#cover-caption {
     width: 100%;
 }
-#submitButtonForm {
```
Comment (sammysosa23) [inline css/styleProject.css]: ignore

File: css/styleProject.css [hunk 6]
```diff
@@ -17,22 +17,137 @@ html, body {
...
-    text-align: center;
```
Comment (sammysosa23) [inline css/styleProject.css]: ignore

Comment (sammysosa23) [inline css/styleProject.css]: ignore

File: css/styleProject.css [hunk 7]
```diff
@@ -17,22 +17,137 @@ html, body {
...
-    align-items: center;
```
Comment (sammysosa23) [inline css/styleProject.css]: ignore

File: css/styleProject.css [hunk 8]
```diff
@@ -17,22 +17,137 @@ html, body {
...
+.form-inline {
+    margin-right: 10px;
 }
-#cover-caption {
```
Comment (sammysosa23) [inline css/styleProject.css]: ignore

File: css/styleProject.css [hunk 9]
```diff
@@ -17,22 +17,137 @@ html, body {
...
-    width: 100%;
```
Comment (sammysosa23) [inline css/styleProject.css]: ignore

File: css/styleProject.css [hunk 10]
```diff
@@ -17,22 +17,137 @@ html, body {
...
+    outline: none;
+    border-radius: 4px;
+    box-shadow: inset 0 0 3px 3px #fafafa;
+}
+/* NAVBAR - ACTIVE - WHO WE ARE - CONTACT US */
+li[class="nav-item"], li[class="nav-item active"] {
+    font-size: 12px;
+}
 
-/* NEED TO ADD LIGHT BOX FOR NAME/EMAIL/SUBMIT */
```
Comment (sammysosa23) [inline css/styleProject.css]: ignore

File: indexProject.html [hunk 11]
```diff
@@ -1,84 +1,171 @@
...
 <html lang="en">
+
 <head>
+    <link rel='shortcut icon' type='image/x-icon' href='./assets/images/favicon-youtube.ico' />
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>*Project One*</title>
 
-    <!-- BOOTSTRAP CSS -->
```
Comment (sammysosa23) [inline indexProject.html]: ignore

File: indexProject.html [hunk 12]
```diff
@@ -1,84 +1,171 @@
...
+    <!--======== BOOTSTRAP CSS =======-->
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css">
-
```
Comment (sammysosa23) [inline indexProject.html]: ignore

File: indexProject.html [hunk 13]
```diff
@@ -1,84 +1,171 @@
...
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>*Project One*</title>
 
-    <!-- BOOTSTRAP CSS -->
+    <!--======== BOOTSTRAP CSS =======-->
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css">
-   
-    <!-- CSS  -->
```
Comment (sammysosa23) [inline indexProject.html]: ignore

File: indexProject.html [hunk 14]
```diff
@@ -1,84 +1,171 @@
...
+    <link src="https://www.gstatic.com/firebasejs/5.7.3/firebase.js"></link>
+    <script src="https://www.gstatic.com/firebasejs/3.4.0/firebase.js"></script>
+
+    <!--======= CSS  =======-->
     <link rel="stylesheet" type="text/css" href="css/styleProject.css">
 
 </head>
+
 <body>
-    <!-- NAVIGATION BAR -->
```
Comment (sammysosa23) [inline indexProject.html]: ignore

File: indexProject.html [hunk 15]
```diff
@@ -1,84 +1,171 @@
...
+    <!--======= NAVIGATION BAR =======-->
     <nav class="navbar navbar-expand-sm navbar-dark bg-dark navbar-full" id="nav-main">
-        <a class="navbar-brand" href="#">Logo</a>
```
Comment (sammysosa23) [inline indexProject.html]: ignore

File: indexProject.html [hunk 16]
```diff
@@ -1,84 +1,171 @@
...
+        <a class="navbar-brand" href=""><img class="logoImage" src="assets/images/ketchup2.png" alt="ketchup splatter"></a>
             <ul class="navbar-nav mr-auto">
                 <li class="nav-item active">
-                    <a class="nav-link" href="#">####<span class="sr-only">(current)</span></a>
```
Comment (sammysosa23) [inline indexProject.html]: ignore

File: indexProject.html [hunk 17]
```diff
@@ -1,84 +1,171 @@
...
-                </li>
```
Comment (sammysosa23) [inline indexProject.html]: ignore

File: indexProject.html [hunk 18]
```diff
@@ -1,84 +1,171 @@
...
-                <li class="nav-item">
```
Comment (sammysosa23) [inline indexProject.html]: ignore

File: indexProject.html [hunk 19]
```diff
@@ -1,84 +1,171 @@
...
-                    <a class="nav-link" href="#">####</a>
```
Comment (sammysosa23) [inline indexProject.html]: ignore

File: css/styleProject.css (hunk 3 above)
Comment (sammysosa23) [inline css/styleProject.css]: ignore

Comment (sammysosa23) [inline css/styleProject.css]: ignore

File: css/styleProject.css (hunk 4 above)
Comment (sammysosa23) [inline css/styleProject.css]: ignore

File: css/styleProject.css [hunk 20]
```diff
@@ -17,22 +17,137 @@ html, body {
     align-items: center;
 
 }
-#trendingButtons.absolute {
-    position: absolute;
```
Comment (sammysosa23) [inline css/styleProject.css]: ignore

File: css/styleProject.css [hunk 21]
```diff
@@ -17,22 +17,137 @@ html, body {
...
-    top: 50px;
```
Comment (sammysosa23) [inline css/styleProject.css]: ignore

File: indexProject.html [hunk 22]
```diff
@@ -1,84 +1,171 @@
...
                             </div>
-                            <br>
                         </div>
                     </div>
                 </div>
             </div>
+        </div>
     </section>
 
-    <!-- INPUT FORM -->
```
Comment (sammysosa23) [inline indexProject.html]: ignore

Comment (sammysosa23) [inline indexProject.html]: ignore

File: indexProject.html [hunk 23]
```diff
@@ -1,84 +1,171 @@
...
+                                    <p class="card-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras sagittis nibh in augue porttitor, vel bibendum mi tristique. Nam cursus tortor sodales, porta neque et, fermentum quam.</p>
+                                    <a href="https://github.com/skendall74" target="_blank">Portfolio</a>
+                                </div>
+                                <div class="card">
+                                    <img src="assets/images/sam.jpg" alt="Sam Terrazas">
+                                    <p class="card-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras sagittis nibh in augue porttitor, vel bibendum mi tristique. Nam cursus tortor sodales, porta neque et, fermentum quam.</p>
+                                    <a href="https://github.com/sammysosa23" target="_blank">Portfolio</a>
+                                </div>
                             </div>
-                            <br>
```
Comment (sammysosa23) [inline indexProject.html]: ignore

File: indexProject.html [hunk 24]
```diff
@@ -1,84 +1,171 @@
...
-                        <h1 class="display-3">Welcome to our "Ketchup"</h1>
-                        <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+                        <h1 class="display-3">Welcome to Ketchup</h1>
+                        <p class="paragraph">Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>      
+                    </div>
+                </div>
+            </div>
+    </section>        
 
-                        <div id="contentContainer">
```
Comment (sammysosa23) [inline indexProject.html]: ignore

File: indexProject.html [hunk 25]
```diff
@@ -1,84 +1,171 @@
...
                 </form>
     </nav>
-    <!-- BODY CONTAINER -->
+    
+    <!--======= BODY CONTAINER =======-->
     <section id="cover">
             <div id="cover-caption">
                 <div class="container">
                     <div class="col-sm-10 offset-sm-1">
-                        <h1 class="display-3">Welcome to our "Ketchup"</h1>
```
Comment (sammysosa23) [inline indexProject.html]: ignore

File: indexProject.html [hunk 26]
```diff
@@ -1,84 +1,171 @@
...
                 </li>
             </ul>
                 <form class="form-inline">
-                    <input class="form-control mr-1" type="search" placeholder="Search" aria-label="Search">
-                    <button class="btn btn-outline-success" type="submit">Search</button>
+                    <input class="form-control mr-1" type="search" placeholder="Search for videos" aria-label="Search">
+                    <button class="btn btn-outline-dark" type="submit"><img class="searchImage" src="assets/images/searchIcon.png" alt="search icon"></button>
                 </form>
     </nav>
-    <!-- BODY CONTAINER -->
```
Comment (sammysosa23) [inline indexProject.html]: ignore

File: indexProject.html [hunk 27]
```diff
@@ -1,84 +1,171 @@
...
-                    <a class="nav-link" href="#">####</a>
+                    <a class="nav-link" href="#whoWeAre">WHO ARE WE</a>
                 </li>
                 <li class="nav-item">
-                    <a class="nav-link" href="#">####</a>
+                    <a class="contactForm nav-link" href="contactUsForm">CONTACT US</a>
                 </li>
             </ul>
                 <form class="form-inline">
-                    <input class="form-control mr-1" type="search" placeholder="Search" aria-label="Search">
```
Comment (sammysosa23) [inline indexProject.html]: ignore

Comment (sammysosa23) [inline indexProject.html]: ignore

File: indexProject.html [hunk 28]
```diff
@@ -1,84 +1,171 @@
...
-                <li class="nav-item">
-                    <a class="nav-link" href="#">####</a>
+                    <a class="nav-link" href="#">HOME PAGE<span class="sr-only">(current)</span></a>
                 </li>
                 <li class="nav-item">
-                    <a class="nav-link" href="#">####</a>
+                    <a class="nav-link" href="#whoWeAre">WHO ARE WE</a>
                 </li>
                 <li class="nav-item">
-                    <a class="nav-link" href="#">####</a>
```
Comment (sammysosa23) [inline indexProject.html]: ignore

File: indexProject.html [hunk 29]
```diff
@@ -1,84 +1,171 @@
...
             <ul class="navbar-nav mr-auto">
                 <li class="nav-item active">
-                    <a class="nav-link" href="#">####<span class="sr-only">(current)</span></a>
-                </li>
-                <li class="nav-item">
-                    <a class="nav-link" href="#">####</a>
+                    <a class="nav-link" href="#">HOME PAGE<span class="sr-only">(current)</span></a>
                 </li>
                 <li class="nav-item">
-                    <a class="nav-link" href="#">####</a>
```
Comment (sammysosa23) [inline indexProject.html]: ignore

File: indexProject.html (hunk 19 above)
Comment (sammysosa23) [inline indexProject.html]: ignore

File: indexProject.html (hunk 18 above)
Comment (sammysosa23) [inline indexProject.html]: ignore

File: indexProject.html (hunk 17 above)
Comment (sammysosa23) [inline indexProject.html]: ignore

**Comment by sammysosa23** [comment]
test

**Comment by skendall74** [comment]
Looks like I'm not able to clean up this branch, guys.  I tried to take a look at it.  I don't have access to other people's branches.  I am taking a look at the master.

**Pull request merged**